### PR TITLE
Add new story after knowing about new developer.

### PR DESCRIPTION
**Pull request by MCorrea92**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1682)

### CODE REVIEW

**Comment by MCorrea92** [comment]
Please review this, @casey-collab

File: portugues/revisa/tenta-consegue/sem-resposta/contato-desenvolvedor.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 Voc� pergunta para o desenvolvedor quando a nova versao do aplicativo estar� dispon�vel para download.
 Por�m ele responde que nao est� mais envolvido com o mesmo.
+
+[A ideia do coco parecia �tima mesmo](coco/muito-quente.md)
+[Voc� vai atr�s do novo desenvolvedor](procura-novo-desenvolvedor/novo-desenvolvedor.md)
```
Comment (casey-collab) [inline portugues/revisa/tenta-consegue/sem-resposta/contato-desenvolvedor.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by MCorrea92** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :melon:

**Pull request merged**